### PR TITLE
Scenicplus

### PR DESCRIPTION
**Pull request by gyear**
For INC1437159 - `adjustText-0.7.3-foss-2022a.eb BioServices-1.11.2-foss-2022a.eb gensim-4.3.2-foss-2022a.eb rapidfuzz-2.13.7-foss-2022a.eb Levenshtein-0.22.0-foss-2022a.eb MACS2-2.2.9.1-foss-2022a.eb panpipes-0.3.1-foss-2022a.eb pyfasta-0.5.2-foss-2022a.eb Sphinx-RTD-Theme-1.3.0-GCCcore-11.3.0.eb scenicplus-1.0.0-foss-2022a.eb`

Based on https://github.com/easybuilders/easybuild-easyconfigs/pull/19060 but updated polars version to match that we have installed. Also removed  osdependencies in Levenshtein install

* [x] Assigned to reviewer

Default:
* [ ] EL8-icelake
* [ ] EL8-cascadelake
* [ ] EL8-haswell

### CODE REVIEW

File: easyconfigs/l/Levenshtein/Levenshtein-0.22.0-foss-2022a.eb
Comment (c-gr34sl3y) [inline easyconfigs/l/Levenshtein/Levenshtein-0.22.0-foss-2022a.eb:1]: Although the software is named Levenshtein as shown in the dev's git and pypi, we've named this `python-Levenshtein` before: https://github.com/bear-rsg/bear-eb/blob/9b701a056c5d3574bc74e7a01a2896ccc11a3462/easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.2-foss-2021b.eb

Upstream have also done the same - https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/python-Levenshtein/

Is it worth leaving it as it has been or reflecting the software's name more accurately?

Comment (gyear) [reply easyconfigs/l/Levenshtein/Levenshtein-0.22.0-foss-2022a.eb:1]: I agree it should be python-Levenshtein I will change it

Comment (gyear) [reply easyconfigs/l/Levenshtein/Levenshtein-0.22.0-foss-2022a.eb:1]: From https://github.com/maxbachmann/python-Levenshtein

> The package was renamed to Levenshtein

**Review by c-gr34sl3y — APPROVED** [review]
lgtm

**Pull request merged**